src/language/languageServerController.ts

```diff
@@ -22,6 +22,12 @@ import { ServerCommands } from './serverCommands';
 
 const log = createLogger('language server controller');
 
+enum CompletionsCache {
+  DATABASES = 'databases',
+  COLLECTIONS = 'collections',
+  FIELDS = 'fields',
+}
```

Comment:
Any particular reason for using an `enum` here?

Reply:
When writing code have suggestions of valid props.